requirements/doc.txt

```diff
@@ -104,7 +104,7 @@ six==1.16.0
     #   edx-sphinx-theme
 snowballstemmer==2.2.0
     # via sphinx
-sphinx==4.5.0
+sphinx==5.0.0
```

Comment:
Some [incompatible changes](https://github.com/sphinx-doc/sphinx/blob/5.x/CHANGES#L43) but not related to the usage in this package.